packages/vitest/src/node/reporters/json.ts

```diff
@@ -10,7 +10,7 @@ import { parseStacktrace } from '../../utils/source-map'
 // the following types are extracted from the Jest repository (and simplified)
 // the commented-out fields are the missing ones
 
-type Status = 'passed' | 'failed' | 'skipped' | 'pending' | 'todo' | 'disabled'
+type Status = 'passed' | 'failed' | 'skipped' | 'pending' | 'todo' | 'disabled' | 'repeated'
```

Comment:
Do we really need another state for this? This is just `run`, isn't it? It should just update `repeats` property.